task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +227,26 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    let resultString = "";
+    let smallerNumber = 0;
+    
+     a > b ? (smallerNumber = b) : (smallerNumber = a , a = b);
+
+    return resultString =`${chekingBrackets(isStartIncluded,"open")}${smallerNumber}, ${a}${chekingBrackets(isEndIncluded,"cloced")}`;
```

Comment:
This is too complex a solution for this simple task.

Reply:
Fixed.